### PR TITLE
Give conspirators a secret radio frequency to communicate in

### PR DESCRIPTION
**Pull request by flrsh**
[FEAT] 

## About the PR 

This PR adds a secret radio frequency for conspirators in the conspiracy gamemode. The frequency is randomized every round and can be accessed with the `:z` prefix, and is added to either the headset the conspirator is wearing at roundstart, or a headset in one of their inventory slots. 

## Why's this needed? 

This makes it easier for conspirators to organize with each other and in general encourages communication. 

## Changelog

```
(u)Flourish:
(*)Conspirators now spawn with a secret radio frequency on their headset. You can access the frequency with the :z prefix to talk with your co-conspirators. 
```

### CODE REVIEW

File: code/datums/gamemodes/conspiracy.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	latejoin_only_if_all_antags_dead = 1 // No hunters until the conspiracy is dead, thanks
 
 	var/maxConspirators = 6
+	var/agent_radiofreq = 1401
```
Comment (ZeWaka) [inline code/datums/gamemodes/conspiracy.dm:8]: why not use the define?

File: code/datums/gamemodes/conspiracy.dm [hunk 2]
```diff
@@ -99,6 +103,18 @@
 	else
 		return candidates
 
+/datum/game_mode/conspiracy/proc/random_radio_frequency()
+	var/f = 0
+	var/list/blacklisted = list(0, 1451, 1457)
+	blacklisted.Add(R_FREQ_BLACKLIST_HEADSET)
+	blacklisted.Add(R_FREQ_BLACKLIST_INTERCOM)
```
Comment (ZeWaka) [inline code/datums/gamemodes/conspiracy.dm:110]: why not just put those all in the list??

File: code/datums/gamemodes/conspiracy.dm [hunk 3]
```diff
@@ -99,6 +103,18 @@
...
+
+	do
+		f = rand(1352, 1439)
+	while (blacklisted.Find(f))
+
+	return f
```
Comment (ZeWaka) [inline code/datums/gamemodes/conspiracy.dm:116]: just use `.` imo

File: code/datums/gamemodes/conspiracy.dm (hunk 1 above)
Comment (flrsh) [reply code/datums/gamemodes/conspiracy.dm:8]: Actually now that I think about it, I don't even use my define anywhere and I don't really need it. Removed!

File: code/datums/gamemodes/conspiracy.dm (hunk 2 above)
Comment (flrsh) [reply code/datums/gamemodes/conspiracy.dm]: Because I copy pasted it!

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**